test/vswhere.test/TestInstance.h

```diff
@@ -112,10 +112,14 @@ class TestInstance :
             const int num_fields = 6;
             SYSTEMTIME st = {};
 
-            if (num_fields == ::swscanf_s(value.c_str(), L"%hd-%hd-%hdT%hd:%hd:%hd", &st.wYear, &st.wMonth, &st.wDay, &st.wHour, &st.wMinute, &st.wSecond))
+            if (num_fields == ::swscanf_s(value.c_str(), L"%hu-%hu-%huT%hu:%hu:%hu", &st.wYear, &st.wMonth, &st.wDay, &st.wHour, &st.wMinute, &st.wSecond))
             {
                 ::SystemTimeToFileTime(&st, pInstallDate);
             }
+            else
+            {
+                hr = E_FAIL;
```

Comment:
This was also meant to fall through. There should be a prefast macro like `__fallthrough()` or something you can tell it this was intended.